src/sentry/release_health/metrics.py

```diff
@@ -376,7 +392,9 @@ def _count_users(total: bool, referrer: str) -> Dict[Any, int]:
         rv = {}
 
         for project_id, release in project_releases:
-            release_tag_value = indexer.resolve(org_id, release)
+            release_tag_value = indexer.resolve(
+                org_id, release, use_case_id=UseCaseKey.RELEASE_HEALTH
```

Comment:
Can we make `us_case_id` always the last arg like it is here but in other calls as well? Seems strange to me that this use_case_id is the first arg passed